tests/ping6_clicker.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2017
+#
+# All rights reserved.
+#
+# This file is distributed under the Clear BSD license.
+# The full text can be found in LICENSE in the root directory.
+
+import rootfs_boot
+from devices import board
+
+class BoardPing6ClickerDev(rootfs_boot.RootFSBootTest):
+	'''Board lowpan interface can ping6 the clicker.'''
+	def runTest(self):
+	# Make Lan-device ping Wan-Device
+		board.sendline('\nping6 -I lowpan0 -c 20 fe80:0000:0000:0000:0204:a30b:0000:c184')
+		board.expect('PING ')
+		board.expect(' ([0-9]+) packets received')
+		board.expect(' ([0-2]+)% packet loss')
+		n = int(board.match.group(1))
+		board.expect(prompt)
+		assert n > 0
```

Comment:
Un-shielded production test JIGs in the lab show in busy environments with hundreds of clickers about a non-zero packet loss isn't always possible. Perhaps we could bump this to 10% to avoid any false negatives?